docs/404.rst

```diff
@@ -0,0 +1,30 @@
+.. SPDX-License-Identifier: MIT
+
+:orphan:
+
+Page Not Found
+--------------
+
+
+We could not find what you were looking for.
+
+
+You might find what you're looking for in one of the below pages:
+
+.. toctree::
+    :maxdepth: 1
+    :titlesonly:
+
+    Index <index>
```

Comment:
These don't link correctly on non-top-level pages, e.g.
https://disnake--797.org.readthedocs.build/en/797/ext/commands/a/b/c/d/stuff

Might be a bug with the extension not updating references in the toc correctly? Not sure.

Reply:
limitation, and not easily solvable. Completely changed the contents in https://github.com/DisnakeDev/disnake/pull/797/commits/61ec6d8592c3ee6536e89a2e03fd1fddfd8d09d6 to make this a nonissue.